packager/packager.livecodescript

```diff
@@ -753,8 +792,11 @@ command packagerBuildStandalones pStandaloneStackFilename, pBuildForPlatform, pB
     end if
 
     # Only build for target platform
+    local tPlatform, tPlatformsToBuildFor
+    put _platformsToBuildFor(tTempStandaloneStack, pBuildForPlatform) into tPlatformsToBuildFor
+
     repeat for each item tPlatform in kStandaloneBuilderPlatforms
-      set the cRevStandaloneSettings[tPlatform] of stack tTempStandaloneStack to tPlatform is pBuildForPlatform
+      set the cRevStandaloneSettings[tPlatform] of stack tTempStandaloneStack to tPlatform is in tPlatformsToBuildFor
```

Comment:
@trevordevore I _think_ I found the issue. Should be
```
 set the cRevStandaloneSettings[tPlatform] of stack tTempStandaloneStack to tPlatform is among the items of tPlatformsToBuildFor
```
I think this resulted in `Windows` and `Windows x86-64` both being built together and causing these issues.

Reply:
Thanks for tracking this down @montegoulding. I'm sorry for the trouble.

Reply:
No problem mate